src/openms/source/FORMAT/HANDLERS/MzIdentMLHandler.cpp

```diff
@@ -986,7 +991,7 @@ namespace OpenMS
                 {
                   if (mods[s].hasSubstring(jt->getMetaValue("xl_mod")))
                   {
-                    ResidueModification mod;
+                    const ResidueModification* mod;
```

Comment:
```suggestion
                    const ResidueModification* mod = nullptr;
```
does that work with `const` ?

Reply:
I always need to look it up https://stackoverflow.com/questions/1143262/what-is-the-difference-between-const-int-const-int-const-and-int-const
works because this means: pointer to a const ResidueModification